concepticondata/conceptlists.tsv

```diff
@@ -215,3 +215,4 @@ Luniewska-2016-299	Łuniewska, Magdalena AND Haman, Ewa AND Armon-Lotem, Sharon
 Gampe-2017-48	Gampe, Anja AND Kurthen, Ira AND Daum, Moritz M.	2017		48	acquisition	English	Global	http://journals.sagepub.com/doi/full/10.1177/0142723717736450	Gampe2017		A list that was developed to assess bilingual children's vocabulary (Swiss German native speakers). The MEAN* columns contain information about the average correctness in the lexical decision task for each item. Items are based on [Luniewska 299](:ref:Luniewska-2016-299).		
 Auracher-2017-16	Auracher, Jan	2017		16	acquisition	English	Global	https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5665516/	Auracher2017		Small list designed for eliciting reaction times to animal pictures. The animal list was included as one experiment (in a total of three different experiments) to differentiate between the reaction times of nonsense words and emotional body postures.		
 Pagel-2018-200	Pagel, Mark AND Meade, Andrew	2018		200	basic	English	Global	http://rstb.royalsocietypublishing.org/highwire/filestream/38234/field_highwire_adjunct_files/0/rstb20160517supp1.pdf	Pagel2018		The list is based on the [210-item list of ABVD](:ref:Blust-2008-210), the  [207-item list of IELex](:ref:Dunn-2012-207), and the [100-item list by Grollemund et al.](:ref:Grollemund-2015-100). The RATE column describes the lexical replacement rate of an item per year, i.e. its lexical stability. Note that the authors state that they took 102 meanings from [Grollemund et al. (2015)](:bib:Grollemund2015), whereas, to the best of our knowledge, the list only contains 100 meanings.		
+Davies-1985-116	Davies, John AND Comrie, Bernard	1985		116	basic,areal	English	New Guinea	http://sealang.net/archives/pl/pdf/PL-A63.275.pdf	Davies1985		A 116 item wordlist survey of the Upper Yuat region. SIL survey list varient		
```

Comment:
I don't understand the last half-sentence. And could you add a full stop after the note ;)

Reply:
done